scripts/ops/verify_supabase_swap.sql

```
@@ -0,0 +1,92 @@
+-- Post-deploy / post-reclaim verification for Capgo-EU swap pressure.
+-- Prefer psql. Avoids unbounded whole-table counts where possible.
+
+SELECT pg_size_pretty(pg_database_size(current_database())::bigint) AS db_size;
+
+SELECT name, setting, unit
```

SQLFluff (4.2.2)

[error] Select targets should be on a new line unless there is only one select target.

(LT09)